dame_flame/data_cleaning.py

```diff
@@ -279,7 +279,15 @@ def process_input_file(df, treatment_column_name, outcome_column_name,
     directly the input file.
     
     '''
-    
+    #Ensuring that no entries are non-integers
```

Comment:
These checks need to be done on both the input and the holdout dataset, please! So do it on both and make sure the notice with the "raise Exception" is clear for users on which one was wrong, please :) Thanks so much!